packages/tradershub/src/components/Dialog/DialogHeader.tsx

```diff
@@ -12,6 +12,7 @@ import CloseIcon from '../../public/images/ic-close-dark.svg';
  */
 type TDialogHeader = {
     className?: string;
+    heading?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
     hideCloseButton?: boolean;
     title?: string;
 };
```

Comment:
```suggestion
const headingSizes: Record<string, React.ReactNode> = {
        h1: Heading.H1,
        h2: Heading.H2,
        h3: Heading.H3,
        h4: Heading.H4,
        h5: Heading.H5,
        h6: Heading.H6,
    };

type TDialogHeader = {
    className?: string;
    heading?: keyof typeof headingSizes;
    hideCloseButton?: boolean;
    title?: string;
};
```

Reply:
why is this necessary ?

Reply:
You dont need to specifically hardcoding the type. Thats why use can use Heading from Reactnode